src/Pages/StartScreen/StartScreenMainGameControls.tsx

```diff
@@ -0,0 +1,40 @@
+import { useNavigate } from "@reach/router";
+import { resetGameState } from "gameHandlers/resetGame";
+import { RoutPaths } from "Pages/routes";
+import { StorageKeys } from "services/getDataFromLocalStorage";
+
+export const StartScreenMainGameControls = () => {
+  const navigate = useNavigate();
+
+  return (
+    <main className="main-content-wrapper flex column align-center justify-center">
+      <div className="flex column">
+        {localStorage.getItem(StorageKeys.CurrentVariants) ? (
```

Comment:
just to know one more option of the writing case if we need to return nothing with some condition will be:
localStorage.getItem(StorageKeys.CurrentVariants) && (
 <button
            onClick={() => {
              navigate(RoutPaths.GameScreen);
            }}
            className="main-game-button"
          >
            Continue Game
          </button>
)